test/index.js

```diff
@@ -560,4 +567,37 @@ describe('Marked renderer', () => {
       '<img src="http://bar.com/b.jpg" alt="a&quot;b" title="c&gt;d"></p>\n'
     ].join('\n'));
   });
+
+  describe('exec filter to extend', () => {
+    it('should render image url to data-src', () => {
+      const ctx = {
+        config: {
+          marked: {}
+        },
+        execFilterSync: (type, filterRenderer) => {
```

Comment:
After `hexo` instance is import to the unit test, you should then register (instead of declare) the filter you want to test.